src/lib/users/dialogs/inst_user_first.rb

```diff
@@ -0,0 +1,661 @@
+# encoding: utf-8
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2006-2012 Novell, Inc. All Rights Reserved.
+#
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, contact Novell, Inc.
+#
+# To contact Novell about this file by physical or electronic mail, you may find
+# current contact information at www.novell.com.
+# ------------------------------------------------------------------------------
+
+# File:	clients/inst_user_first.ycp
+# Package:	Configuration of users and groups
+# Summary:	Dialog for creating the first user during installation
+# Authors:     Jiri Suchomel <jsuchome@suse.cz>
+#
+# $Id$
+require "yast"
+require "ui/event_dispatcher"
+require "users/dialogs/users_to_import"
+require "users/ca_password_validator"
+require "users/local_password"
+
+module Yast
+  class InstUserFirstDialog
+    include ::UI::EventDispatcher
```

Comment:
In its current form, it is really `InstallationDialog`, since it uses `Yast::GetInstArgs`, makes `accept_handler` kind of an alias for `next_handler` and that kind of installation specific stuff.